bot/exts/evergreen/githubinfo.py

```diff
@@ -74,24 +83,71 @@ async def get_github_info(self, ctx: commands.Context, username: Optional[str])
 
                 embed.add_field(name="Followers",
                                 value=f"[{user_data['followers']}]({user_data['html_url']}?tab=followers)")
-                embed.add_field(name="\u200b", value="\u200b")
                 embed.add_field(name="Following",
                                 value=f"[{user_data['following']}]({user_data['html_url']}?tab=following)")
 
             embed.add_field(name="Public repos",
                             value=f"[{user_data['public_repos']}]({user_data['html_url']}?tab=repositories)")
-            embed.add_field(name="\u200b", value="\u200b")
 
             if user_data['type'] == "User":
                 embed.add_field(name="Gists", value=f"[{gists}](https://gist.github.com/{username})")
 
                 embed.add_field(name=f"Organization{'s' if len(orgs)!=1 else ''}",
                                 value=orgs_to_add if orgs else "No organizations")
-                embed.add_field(name="\u200b", value="\u200b")
             embed.add_field(name="Website", value=blog)
 
         await ctx.send(embed=embed)
 
+    @github_group.command(name='repo', aliases=('repository',))
+    async def github_repo_info(self, ctx: commands.Context, repo: Optional[str]) -> None:
+        """
+        Fetches a repositories's GitHub information. Repository should look like `user/reponame`.
+
+        Repository is optional and sends the help command if not specified.
+        """
+        if repo is None:
```

Comment:
You can just make `repo` a required argument and then you can skip this step, it would automatically be handled by the error handler.